paper-toast.html

```diff
@@ -187,6 +187,8 @@
          * Called when the value of `opened` changes.
          */
         _openedChanged: function() {
+          this.cancelAsync(this._autoClose);
+          this._autoClose = undefined;
```

Comment:
is there any advantage in setting it to `null` (e.g. performance)?
I was tempted to do
```javascript
this._autoClose = undefined;
delete this._autoClose;